apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/disable-trash-template/index.js

```diff
@@ -0,0 +1,31 @@
+/* global fullSiteEditing */
+
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { select, subscribe } from '@wordpress/data';
+
+domReady( () => {
```

Comment:
We can hide this much more easily with just CSS:

```css
.post-type-wp_template .editor-post-trash {
  display: none;
}
```

Reply:
Note that we can more cleanly override actions too, see https://github.com/Automattic/wp-calypso/pull/34855/files

Maybe a combo of display none, and adding a no-op for trashPost for this CPT?

Reply:
Updated on a5cea6b77166fc59577ef8da014a4208f8b59cad.